internal/supervisor/installation.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/mattermost/mattermost-cloud/internal/webhook"
 	"github.com/mattermost/mattermost-cloud/model"
 	mmv1alpha1 "github.com/mattermost/mattermost-operator/pkg/apis/mattermost/v1alpha1"
+	"github.com/pkg/errors"
```

Comment:
I've been putting third party imports in a different section from the first party ones, e.g.

stdlib imports

third party imports

first party imports

Now, I _thought_ this was already a part of our style guidelines but it's absolutely possible that I imagined it, because this is an old habit of mine anyway. That said, if you're on board with that style, I'd like to see the imports follow that pattern here.

Reply:
Yeah, that is absolutely what I try to do as well. My editor added this and I didn't review it. I will fix it shortly.